{"uuid": "9c307fd5-c488-4112-a665-fc732066afad", "children": ["e0779f27-2dec-463a-8ba6-07d9db35d0c9"], "befores": [{"name": "setup", "status": "passed", "start": 1754556376410, "stop": 1754556394180}], "afters": [{"name": "setup::0", "status": "broken", "statusDetails": {"message": "KeyboardInterrupt\n", "trace": "  File \"C:\\Users\\MITHU\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\allure_commons\\_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"C:\\Users\\MITHU\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\_pytest\\fixtures.py\", line 911, in _teardown_yield_fixture\n    next(it)\n  File \"G:\\Work Station For Appium Testing\\ArcOne Mobile App Automation Testing\\conftest.py\", line 15, in setup\n    driver.quit()\n  File \"C:\\Users\\MITHU\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 612, in quit\n    self.execute(Command.QUIT)\n  File \"C:\\Users\\MITHU\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 451, in execute\n    response = cast(RemoteConnection, self.command_executor).execute(driver_command, params)\n  File \"C:\\Users\\MITHU\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 404, in execute\n    return self._request(command_info[0], url, body=data)\n  File \"C:\\Users\\MITHU\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 428, in _request\n    response = self._conn.request(method, url, body=body, headers=headers, timeout=self._client_config.timeout)\n  File \"C:\\Users\\MITHU\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\urllib3\\_request_methods.py\", line 135, in request\n    return self.request_encode_url(\n  File \"C:\\Users\\MITHU\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\urllib3\\_request_methods.py\", line 182, in request_encode_url\n    return self.urlopen(method, url, **extra_kw)\n  File \"C:\\Users\\MITHU\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\urllib3\\poolmanager.py\", line 459, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n  File \"C:\\Users\\MITHU\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\urllib3\\connectionpool.py\", line 787, in urlopen\n    response = self._make_request(\n  File \"C:\\Users\\MITHU\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\urllib3\\connectionpool.py\", line 534, in _make_request\n    response = conn.getresponse()\n  File \"C:\\Users\\MITHU\\AppData\\Local\\Programs\\Python\\Python310\\lib\\site-packages\\urllib3\\connection.py\", line 565, in getresponse\n    httplib_response = super().getresponse()\n  File \"C:\\Users\\MITHU\\AppData\\Local\\Programs\\Python\\Python310\\lib\\http\\client.py\", line 1374, in getresponse\n    response.begin()\n  File \"C:\\Users\\MITHU\\AppData\\Local\\Programs\\Python\\Python310\\lib\\http\\client.py\", line 318, in begin\n    version, status, reason = self._read_status()\n  File \"C:\\Users\\MITHU\\AppData\\Local\\Programs\\Python\\Python310\\lib\\http\\client.py\", line 279, in _read_status\n    line = str(self.fp.readline(_MAXLINE + 1), \"iso-8859-1\")\n  File \"C:\\Users\\MITHU\\AppData\\Local\\Programs\\Python\\Python310\\lib\\socket.py\", line 705, in readinto\n    return self._sock.recv_into(b)\n"}, "start": 1754556406726, "stop": 1754556408090}], "start": 1754556376410, "stop": 1754556408090}